hyperparameter_tuning/huggingface_multiclass_text_classification_20_newsgroups/hpo_huggingface_text_classification_20_newsgroups.ipynb

```diff
@@ -0,0 +1,1363 @@
+{
```

Comment:
<p>repeated cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2931/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
This is not a repeated cell. Note that I am reading from `rec.motorcycles` show `df["text"][0]` and then read from `comp.sys.mac.hardware` , and show `df["text"][0]`